### ISSUE TITLE
pinterest.com is reporting taken for usernames which are not taken

### ISSUE THREAD

**Issue by jayvdb** [manu-chroma/username-availability-checker#17]

**Comment by manu-chroma** [comment]
Unfortunately, HTTP code is 200 for both url having taken/available usernames. 
``` python
In [1]: import requests as r
In [2]: url_taken = 'https://in.pinterest.com/manvendra0310/'
In [3]: res = r.get(url_taken)
In [4]: res.status_code
Out[4]: 200

In [5]: res.history
Out[5]: []

In [6]: url_available = 'https://in.pinterest.com/manvendraaaa/'
In [7]: res = r.get(url_available)
In [8]: res.status_code
Out[8]: 200

In [9]: res.history
Out[9]: [<Response [302]>]
``` 

I'm implementing a hacky solution for the time being and should fix the issue for now.

**Comment by manu-chroma** [comment]
Changes should reflect on the website now. Also, there is improvement in the front end too as well as basic regex in the input filed on webpage.

**Comment by manu-chroma** [comment]
@jayvdb which one is more fault tolerant in your opinion?

``len(res.history) > 0`` or  ``bytes(username, encoding='utf-8') in res.content``

**Comment by jayvdb** [comment]
I think `bytes(username, encoding='utf-8') in res.content` is the more stable & re-usable algorithm.

**Comment by manu-chroma** [comment]
Addressed in: https://github.com/manu-chroma/username-availability-checker/commit/0acde56bf11e28326f1a4a6e3e8e40493c448e99

Closing.
